### ISSUE TITLE
Runner Results Export function should Export CSV, PDF as a proper Report

### ISSUE THREAD

**Issue by Crheederster** [a85/POSTMan-Chrome-Extension#4172]
##Description:
The runner results export function, exports the results as a json file which serves no purpose.  As a test analyst I would expect the export function in this case, to export the results in a readable, useful format like Excel. 

## App Details:
Postman for Windows
Version 5.5.2
win32 10.0.15063 / x64

## Issue Report:
 1. Did you encounter this recently, or has this bug always been there: I have just encountered this - 12-02-2018
 2. Expected behavior: When I have just run a collection of tests using the Runner function and I view the results summary, I would expect the export function to export a readable Results summary. 
3. Current behavior: When I select the export function a json file/packet is exported. It serves no purpose. 
4. Screenshots (if applicable) - Attached:

![1_export results](https://user-images.githubusercontent.com/33580608/36104262-267ef984-101a-11e8-8424-6652befc21de.png)
![2_export results](https://user-images.githubusercontent.com/33580608/36104263-26b5804e-101a-11e8-8890-59b63db4dbe0.PNG)
![3_export results](https://user-images.githubusercontent.com/33580608/36104351-71eadeb0-101a-11e8-9118-6f9818a21729.PNG)

**Comment by Chris-Yarland-Price** [comment]
I totally agree and this should be a better way of doing things currently what we do is in the runner results if you ctrl + A, ctrl + C and paste into a txt file it gives you a readable format it is still way too verbose but it is better than the alternatives. especially for adding into JIRA as proof of testing

**Comment by sivcan** [comment]
Hi @Crheederster - Thanks for this suggestion. I'll be putting it up as a feature request.

**Comment by Crheederster** [comment]
Thank You for the feedback :)

**Comment by ykhodos** [comment]
+1

**Comment by stixaw** [comment]
+1

**Comment by m1kjay** [comment]
+1 thanks

**Comment by GaboBDCP** [comment]
+1

**Comment by matthew-macartney** [comment]
+1

**Comment by msignataur** [comment]
+1

**Comment by KristofVanKriekingen1337** [comment]
+1

**Comment by nebloc** [comment]
+1

**Comment by bjones01001101** [comment]
+1

**Comment by maulmota** [comment]
+1

**Comment by murtuzak** [comment]
+1

**Comment by DammyDev** [comment]
+1

**Comment by kavanozkafa** [comment]
+1

**Comment by pbreit** [comment]
+1

But CSV, not Excel.

https://www.npmjs.com/package/json2csv
